install/ansible/install.sh

```diff
@@ -131,6 +136,9 @@ cp /var/contiv/key.pem /ansible/roles/auth_proxy/files/
 if [ "$aci_image" != "" ]; then
 	sed -i.bak "s#.*aci_gw_image.*#\"aci_gw_image\":\"$aci_image\",#g" "$env_file"
 fi
+if [ "$contiv_v2plugin_install" == "true" ]; then
+	sed -i.bak "s#.*contiv_v2plugin_install.*#\"contiv_v2plugin_install\":\"True\",#g" "$env_file"
+fi
```

Comment:
Opened https://github.com/contiv/install/issues/149 to handle resetting this for the non-v2 plugin case.
Rest of it LGTM.